pkg/api/dtos/index.go

```diff
@@ -31,4 +31,5 @@ type IndexViewData struct {
 	LoadingLogo             template.URL
 	// Nonce is a cryptographic identifier for use with Content Security Policy.
 	Nonce string
+	Env   string
```

Comment:
Please try to add `// @PERCONA` when modifying grafana code (either TS/GO). It makes it easier to handle conflicts when doing grafana upgrades.